dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml

```diff
@@ -39,25 +40,27 @@
     Attached to this email, you will also find a personalized certificate acknowledging your successful
     completion of Code.org's CS Fundamentals workshop.
 
-  %h3
-    For a limited time
-  %p
-    Code.org is offering $50 off
-    = link_to('Dash robots', 'https://store.makewonder.com/') + '.'
-    These hands-on learning tools reinforce CS concepts taught in Code.org's CS Fundamentals curriculum,
-    like events and loops.
-    = link_to('Learn more', 'https://store.makewonder.com/') + '.'
-    %br
-    %i
-      Valid June 1st-30th, 2016 only. Limited to first 1,000 people.
+  - if @dash_code
+    %h3
+      For a limited time
+    %p
+      Code.org is offering $50 off
+      = link_to('Dash robots', 'https://store.makewonder.com/') + '.'
+      These hands-on learning tools reinforce CS concepts taught in Code.org's CS Fundamentals curriculum,
+      like events and loops.
+      = link_to('Learn more', 'https://store.makewonder.com/') + '.'
+      %br
+      %i
+        Valid June 1st-30th, 2016 only. Limited to first 1,000 people.
```

Comment:
Merged staging to get changes from https://github.com/code-dot-org/code-dot-org/pull/9635, and added that text.